.github/workflows/nix-macos.yml

```diff
@@ -27,8 +27,8 @@ jobs:
       - run: cachix use iohk
       - run: |
           nix-build -j 2 -A haskellPackages.libtorch-ffi.checks.spec
-          nix-build -j 2 -A haskellPackages.hasktorch.checks.spec
-          nix-build -j 2 -A haskellPackages.examples.checks.spec
+          nix-build -j 1 -A haskellPackages.hasktorch.checks.spec
+          nix-build -j 1 -A haskellPackages.examples.checks.spec
```

Comment:
Before disassembling FunctionalSpec, OSkill happened  with `-j1`. `-j1`  intends to reduce memory. I'm not sure why OSkill happened.